Firebase/Core/Private/FIRComponentContainer.h

```diff
@@ -40,7 +41,7 @@ NS_SWIFT_NAME(FirebaseComponentContainer)
 
 /// Register a class to provide components for the interoperability system. The class should conform
 /// to `FIRComponentRegistrant` and provide an array of `FIRComponent` objects.
-+ (void)registerAsComponentRegistrant:(Class)klass;
++ (void)registerAsComponentRegistrant:(id<FIRLibrary>)klass;
```

Comment:
Does this need to be a public API anymore? I think since users are calling `registerLibrary`, we can move this to the internal header for use by the `registerLibrary`call and unit tests.

Reply:
Done. Still need to make the Internal headers really internal though.